cmr/queries.py

```diff
@@ -427,10 +479,10 @@ def revision_date(
         return self
 
     def temporal(
-        self,
-        date_from: Optional[DateLike],
-        date_to: Optional[DateLike],
-        exclude_boundary: bool = False,
+            self,
+            date_from: Optional[DateLike],
+            date_to: Optional[DateLike],
+            exclude_boundary: bool = False,
```

Comment:
I think extra indentation slipped in via another PR:

```suggestion
        self,
        date_from: Optional[DateLike],
        date_to: Optional[DateLike],
        exclude_boundary: bool = False,
```